content/rc/administration/setup/customize-flexible-plan.md

```diff
@@ -0,0 +1,105 @@
+---
+Title: Customize a Flexible plan
+description:
+weight: 30
+alwaysopen: false
+categories: ["RC"]
+aliases: /rc/administration/customize-pro/
+---
+
+When you create a subscription with a Flexible plan, you customize it to your business needs.
+
+This means you:
+
+- Define deployment settings for your subscription, such as cloud provider, network settings, and so on
+
+- Create at least one database in order to determine cluster size.
+
+- Review the cost estimate for your selections before creating the subscription.
+
+Here's what the process looks like when you start:
+
+&lt;image here>
+
+## Setup settings
+
+The Setup tab specifies general settings for the deployment of your subscription.
```

Comment:
```suggestion
The **Setup** tab specifies general settings for the deployment of your subscription.
```

Reply:
I'm not sure if tabs follow the same rules as buttons as far as how we style the text.

Reply:
Me either, but it makes sense to do so, as I'm referring to specific text on the screen.